src/test/java/seedu/address/logic/commands/DeleteApptCommandTest.java

```diff
@@ -0,0 +1,167 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_APPOINTMENT_DATE_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_APPOINTMENT_TIMEPERIOD_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_NRIC_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalPersons.ALICE;
+import static seedu.address.testutil.TypicalPersons.AMY;
+import static seedu.address.testutil.TypicalPersons.BOB;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.Messages;
+import seedu.address.logic.parser.DeleteApptCommandParserTest;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.appointment.Appointment;
+import seedu.address.model.person.AppointmentExistsPredicate;
+import seedu.address.model.person.Nric;
+import seedu.address.model.person.NricMatchesPredicate;
+import seedu.address.model.person.Person;
+import seedu.address.testutil.PersonBuilder;
+
+/**
+ * Contains the integration tests (interaction with the Model) and unit tests for DeleteApptCommand.
+ */
+
+public class DeleteApptCommandTest {
+
+    @Test
+    public void execute_allFieldsSpecified_success() {
+        Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        Person targetPerson = model.getFilteredPersonList().get(0);
+        Nric nric = targetPerson.getNric();
+        Appointment apptToDelete = targetPerson.getAppointments()
+                                               .stream()
+                                               .findFirst()
+                                               .orElseThrow(() -> new AssertionError(
+                                                       DeleteApptCommandParserTest.class
+                                                       + ": PLEASE USE ANOTHER TARGET")
```

Comment:
Just a nit: Could be possible to use `Supplier` to abstract out  
`() -> new AssertionError(DeleteApptCommandParserTest.class
                                                       + ": PLEASE USE ANOTHER TARGET"`
                                                       
                                                       
But test cases for this class is comprehensive 👍🏻